types/artifacts.d.ts

```diff
@@ -23,6 +23,10 @@ declare global {
       fetchTime: string;
       /** A set of warnings about unexpected things encountered while loading and testing the page. */
       LighthouseRunWarnings: string[];
+      /** Whether the page was loaded on either a real or emulated mobile device */
+      IsMobile: boolean;
```

Comment:
we might need to bikeshed on these. When I see `artifacts.isMobile`, my first thought is "is what mobile?" :)

It'll make a little more sense when nested in an environment artifact, but do we have other ideas?

(I also like reusing `host` for the UA string artifacts, but (super nitpicky) dislike the `host` prefix vs suffix :)

Reply:
`IsMobileFormFactor`?